Makefile

```diff
@@ -67,9 +71,10 @@ bin/%: cmd/% FORCE
 	$Q $(GO) build -mod=vendor $(LDFLAGS) -v -o $@ ./$<
 
 .PHONY: apply
-apply: | $(KO) $(KUSTOMIZE) ; $(info $(M) ko apply on $(TARGET)) @ ## Apply config to the current cluster
-	$Q $(KUSTOMIZE) build config/$(TARGET) | $(KO) apply $(PLATFORM) -f -
-
+apply: | $(KO) $(KUSTOMIZE) ;
+	$Q ./release.sh $(TARGET) $(PIPELINES) $(TRIGGERS)|| exit ;
```

Comment:
Could we move the script in `./hack` ?

Reply:
We also need to have this for `resolve`, so most likely making this its own target and mark `apply` and `resolve` to be dependencies of it.

Reply:
yep 👍🏽